build.sbt

```diff
@@ -12,6 +13,7 @@ lazy val twitterServerVersion = "18.7.0"
 lazy val circeVersion = "0.9.3"
 lazy val circeIterateeVersion = "0.9.0"
 lazy val catbirdVersion = "18.6.0"
```

Comment:
I see imports are ordered alphabetically, agree on sorting these versions as well? or leave as is?

Reply:
Don't have a strong opinion - feel free to sort if you feel like it.